lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost/Fixture.hs

```diff
@@ -1,10 +1,13 @@
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE LambdaCase #-}
 
 module Cardano.Wallet.Shelley.Network.Blockfrost.Fixture where
 
 import Prelude
 
+import Cardano.Address.Style.Shelley
+    ( unsafeFromRight )
```

Comment:
That's a really weird place to import this from — the package is `cardano-addresses`. 🤔 

`Data.Either` has `fromRight :: b -> Either a b -> b` at least.

Reply:
Ok